pkgs/development/python-modules/clickgen/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pillow
+, libX11
+, libXcursor
+, libpng
+, python
+}:
+
+buildPythonPackage rec {
+  pname = "clickgen";
+  version = "1.1.9";
+
+  src = fetchFromGitHub {
+    owner = "ful1e5";
+    repo = "clickgen";
+    rev = "v${version}";
+    sha256 = "108f3sbramd3hhs4d84s3i3lbwllfrkvjakjq4gdmbw6xpilvm0l";
+  };
+
+  buildInputs = [ libXcursor libX11 libpng ];
+
+  propagatedBuildInputs = [ pillow ];
+
+  pythonImportsCheck = [ "clickgen" ];
+
+  postInstall = ''
+    install -m644 clickgen/xcursorgen.so $out/${python.sitePackages}/clickgen/xcursorgen.so
+  '';
+
+  # Tests fail with ${bitmap_error}.
+  # Application works normally if tests are disabled
+  doCheck = false;
```

Comment:
With this patch the test seems to work fine:

```
diff --git a/pkgs/development/python-modules/clickgen/default.nix b/pkgs/development/python-modules/clickgen/default.nix
index 141fd4569b5..a85cee778b0 100644
--- a/pkgs/development/python-modules/clickgen/default.nix
+++ b/pkgs/development/python-modules/clickgen/default.nix
@@ -6,6 +6,7 @@
 , libXcursor
 , libpng
 , python
+, pytestCheckHook
 }:

 buildPythonPackage rec {
@@ -29,9 +30,7 @@ buildPythonPackage rec {
     install -m644 clickgen/xcursorgen.so $out/${python.sitePackages}/clickgen/xcursorgen.so
   '';

-  # Tests fail with ${bitmap_error}.
-  # Application works normally if tests are disabled
-  doCheck = false;
+  checkInputs = [ pytestCheckHook ];

   meta = with lib; {
     homepage = "https://github.com/ful1e5/clickgen";
```

Reply:
Ok! That's work fine